src/_locales/de/messages.json

```diff
@@ -23,6 +27,18 @@
         "message": "Benachrichtigung anzeigen wenn Titel nicht von der Erweiterung erkannt wird",
         "description": "Option title"
     },
+    "optionScrobblePodcasts": {
+        "message": "Scrobble podcasts",
```

Comment:
Not a fan of this "default" behaviour to pre-fill missing translations with src lang as it might encourage people to modify on here vs on the Transflix UI

Reply:
doesnt seem ideal yeah

Reply:
Fixed with the script